src/vue/forms/SubmitTradeOrderForm.vue

```diff
@@ -0,0 +1,128 @@
+<template>
```

Comment:
it mostly copies CreateTradeOrderForm.vue. let’s think about removing code repeat now

Reply:
Ya, I thought about moving it to some mixin. We can move only two methods and several imports. The template has greater more than one-two diffs. So I have no idea how to create common effective mixin or component but at the same time keep it clean and easy to understand and test.